scripts/install-go2rtc-preview.sh

```diff
@@ -0,0 +1,221 @@
+#!/bin/bash
+
+GO2RTC_VERSION="v1.8.6-4"
+YAML_STREAM="dslr: exec:gphoto2 --capture-movie --stdout#killsignal=sigint"
+CAPTURE_CMD="gphoto2"
+CAPTURE_ARGS="--set-config output=Off --capture-image-and-download --filename=\$1"
+NOTE="don't forget to add --filename=%s."
+
+function info {
+    echo -e "\033[0;36m${1}\033[0m"
+}
+
+function error {
+    echo -e "\033[0;31m${1}\033[0m"
+}
+
+if [ "$UID" != 0 ]; then
+    error "ERROR: Only root is allowed to execute the installer. Forgot sudo?"
+    exit 1
+fi
+
+#Param 1: Question / Param 2: Default / silent answer
+function ask_yes_no {
+    read -p "${1}: " -n 1 -r
+}
+
+function mjpeg_preview() {
+    local arch
+    local goarch
+    local os
+    local file
+
+    if ! command -v go2rtc &>/dev/null || [[ ! $(go2rtc -version) =~ $GO2RTC_VERSION ]]; then
+        info "### Installing go2rtc (version: ${GO2RTC_VERSION})"
+
+        if [[ "$OSTYPE" =~ linux ]]; then
+            os=linux
+        elif [[ "$OSTYPE" =~ darwin ]]; then
+            os=darwin
+        elif [[ "$OSTYPE" =~ cygwin|mysys|win32 ]]; then
+            os=windows
+        else
+            error "### $OSTYPE not supported"
+            exit 1
+        fi
+
+        arch=$(uname -m)
+        if [[ "$arch" == "x86_64" ]]; then
+            goarch="amd64"
+        elif [[ "$arch" == "i386" ]]; then
+            goarch="386"
+        elif [[ "$arch" == "armv7l" ]]; then
+            goarch="armv7"
+        elif [[ "$arch" == "armv6l" ]]; then
+            goarch="armv6"
+        elif [[ "$arch" == "aarch64" ]]; then
+            goarch="arm64"
+        else
+            error "### $arch not supported"
+            exit 1
+        fi
+
+        if [[ ! -d /usr/local/bin ]]; then
+            mkdir -p /usr/local/bin
+        fi
+        file="go2rtc_${os}_${goarch}.tar.gz"
+        wget -P /tmp "https://github.com/dadav/go2rtc/releases/download/${GO2RTC_VERSION}/${file}"
```

Comment:
We should be able to use the official repo by now! They merged my changes :)

Reply:
Great! I'll pick your changes and push it here too. Suggestions about switching from "dslr" stream to something more common?